sky/provision/kubernetes/utils.py

```diff
@@ -678,7 +685,7 @@ def check_tpu_fits(candidate_instance_type: 'KubernetesInstanceType',
                        f'{tpu_list_in_cluster_str}. Note that multi-host TPU '
                        'podslices are currently not unsupported.')
 
-    nodes = get_kubernetes_nodes(context)
+    nodes = get_kubernetes_nodes(context=context)
```

Comment:
We can make `get_kubernetes_nodes`'s definition to be the following, so that the positional arguments is disallowed for the context : )

```python
def get_kubernetes_nodes(*, context: Optional[str] = None) -> List[Any]:
```

Reply:
Great idea!